examples/basic/yarn.lock

```diff
@@ -2571,6 +2571,11 @@ emitter-listener@^1.0.1, emitter-listener@^1.1.1:
   dependencies:
     shimmer "^1.2.0"
 
+eme-encryption-scheme-polyfill@^2.0.1:
```

Comment:
I don't see any change in package.json, do you have an idea this this package has been added ?

Reply:
Looks like some merge conflict issue, I removed it 👍